.azure-pipelines.yml

```diff
@@ -0,0 +1,206 @@
+        rm flutter_linux_$FLUTTER_VERSION.tar.xz
+        export PATH="$PATH:$FLUTTER_HOME/bin:$FLUTTER_HOME/bin/cache/dart-sdk/bin"
+        flutter config --no-analytics
+        flutter doctor
+
+        # Install git-repo.
+        mkdir ~/bin
+        export PATH=$PATH:~/bin
+        curl https://storage.googleapis.com/git-repo-downloads/repo > ~/bin/repo
+        chmod a+x ~/bin/repo
+
+        # Install Go Dep.
+        mkdir -p ~/go/bin
+        export PATH=$PATH:~/go/bin
+        curl https://raw.githubusercontent.com/golang/dep/v$GO_DEP_VERSION/install.sh | sh
+
+        # Update PATH for next steps.
+        echo "##vso[task.setvariable variable=path;]$PATH"
+      condition: eq( variables['Agent.OS'], 'Linux' )
+      displayName: Install Linux requirements
+
+    # TODO: Add a step to run "reuse lint" (only on Linux?).
+
+    # Windows requirements.
+    - pwsh: |
+        # Install Chocolatey packages.
+        cinst dep --version $env:GO_DEP_VERSION -y --no-progress
+        cinst haskell-stack --version $env:STACK_VERSION -y --no-progress
+        refreshenv
+
+        # Install CVS. Disabled because msys2 installation takes too long.
+        #cinst msys2 --params "/InstallDir=C:/msys64" --no-progress
+        #C:\msys64\usr\bin\bash -lc "pacman --noconfirm -Sy cvs"
+        #$env:PATH += ";C:\msys64\usr\bin"
+
+        # Install NPM packages.
+        npm install -g bower@$env:BOWER_VERSION
+
+        # Install Python packages.
+        pip install --user conan==$env:CONAN_VERSION pipenv==$env:PYTHON_PIPENV_VERSION reuse virtualenv==$env:VIRTUALENV_VERSION
+        pip show conan
+        conan user # Create the conan data directory. Automatic detection of your arch, compiler, etc.
+
+        # Install Ruby packages.
+        gem install bundler -v $env:BUNDLER_VERSION
+
+        # Install Flutter.
+        Invoke-WebRequest -Uri "https://storage.googleapis.com/flutter_infra/releases/stable/windows/flutter_windows_$env:FLUTTER_VERSION.zip" -OutFile "C:\flutter_windows_$env:FLUTTER_VERSION.zip"
+        7z x C:\flutter_windows_$env:FLUTTER_VERSION.zip -oC:\
+        Remove-Item "C:\flutter_windows_$env:FLUTTER_VERSION.zip"
+        $env:PATH += ";$env:FLUTTER_HOME\bin;$env:FLUTTER_HOME\bin\cache\dart-sdk\bin"
+        flutter config --no-analytics
+        flutter doctor
+
+        # Stop adb because it otherwise locks the working directory which fails the checkout step.
+        Stop-Process -Name "adb"
+
+        ## Install git-repo.
+        Invoke-WebRequest -Uri "https://storage.googleapis.com/git-repo-downloads/repo" -OutFile "$env:PROGRAMFILES\Git\usr\bin\repo"
+
+        # Update PATH for next steps.
+        echo "##vso[task.setvariable variable=path;]$env:PATH"
+      condition: eq( variables['Agent.OS'], 'Windows_NT' )
+      displayName: Install Windows requirements
+      errorActionPreference: continue
+      failOnStderr: false
+
+    # Clone repository.
+    - checkout: self
+      fetchDepth: 1
```

Comment:
Just FYI, this might cause problems in the future when inferring the version from the last Git tag.